common/src/chart.ts

```diff
@@ -0,0 +1,36 @@
+import { AxisScale } from 'd3-axis'
```

Comment:
Refactoring chart types or moving around existing bet-fetching code should be a different PR.
It's good to separate refactors from changes in functionality.